pkg/apis/alerting/v1alpha2/register.go

```diff
@@ -0,0 +1,1380 @@
+		Consumes(restful.MIME_JSON, constants.MIME_MERGEPATCH).
+		Produces(restful.MIME_JSON)
+
+	ws.Route(ws.GET("/namespaces/{ns_name}/pods/{pod_name}/containers/alert").To(alerting.DescribeAlertDetailsContainer).
+		Doc("Describe Alert Details Container Level").
+		Param(ws.PathParameter("ns_name", "Specify namespace").DataType("string").Required(true).DefaultValue("")).
+		Param(ws.PathParameter("pod_name", "Specify pod").DataType("string").Required(true).DefaultValue("")).
+		Param(ws.QueryParameter("search_word", "Specify search word").DataType("string").Required(false)).
+		Param(ws.QueryParameter("alert_ids", "Specify alert ids to query, comma-separated, eg. al-WgBBMmRv3rMP,al-QVjOxnkD3mwW.").DataType("string").Required(false)).
+		Param(ws.QueryParameter("alert_names", "Specify alert names to query, comma-separated, eg. alert-1,alert-2.").DataType("string").Required(false)).
+		Param(ws.QueryParameter("disables", "Specify alert disabled status, comma-separated, eg. true,false.").DataType("string").Required(false)).
+		Param(ws.QueryParameter("running_status", "Specify alert running status, comma-separated, eg. adding,running,deleting,updating,migrating.").DataType("string").Required(false)).
+		Param(ws.QueryParameter("policy_ids", "Specify policy ids to query, comma-separated, eg. pl-zZmOyYqqx7vo,pl-zZPP12kqx7vo.").DataType("string").Required(false)).
+		Param(ws.QueryParameter("creators", "Specify creators to query, comma-separated, eg. admin,user1.").DataType("string").Required(false)).
+		Param(ws.QueryParameter("rs_filter_ids", "Specify resource filter ids to query, comma-separated, eg. rf-ZyzVP265N3l5,rf-zZ416xNqx7vo.").DataType("string").Required(false)).
+		Param(ws.QueryParameter("executor_ids", "Specify alert executor ids to query, comma-separated, eg. alerting-executor-5f8d9bb8b9-4rl95,alerting-executor-5f8d9bb8b9-jw728.").DataType("string").Required(false)).
+		Param(ws.QueryParameter("sort_key", "Sort key. One of t1.alert_id, t1.alert_name, t1.disabled, t1.running_status, t1.alert_status, t1.create_time, t1.update_time, t1.policy_id, t1.rs_filter_id, t1.executor_id, t2.policy_id, t2.policy_name, t2.policy_description, t2.policy_config, t2.creator, t2.available_start_time, t2.available_end_time, t2.create_time, t2.update_time, t2.rs_type_id, t3.rs_filter_id, t3.rs_filter_name, t3.rs_filter_param, t3.status, t3.create_time, t3.update_time, t3.rs_type_id, t4.rs_type_id, t4.rs_type_name, t4.rs_type_param, t4.create_time, t4.update_time, t5.action_id, t5.action_name, t5.trigger_status, t5.trigger_action, t5.create_time, t5.update_time, t5.policy_id, t5.nf_address_list_id.").DataType("string").Required(false)).
+		Param(ws.QueryParameter("reverse", "Sort order, true-desc, false-asc.").DataType("bool").DefaultValue("false").Required(false)).
+		Param(ws.QueryParameter("offset", "Beginning index of result to return. Use this option together with limit.").DataType("uint32").Required(false)).
```

Comment:
For API doc, should we use the same explaination for the same parameters as in notification?
In nf is ://beginning data offset of this page, default 0